### PR TITLE
Integrated intra-shard mini-blocks

### PR DESCRIPTION
**Pull request by miiu96**
- Index in the `Elasticseach` database also the `miniblocks` that are nod included in the `Body` structure

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	github.com/elastic/go-elasticsearch/v7 v7.12.0
 	github.com/multiversx/mx-chain-communication-go v1.0.3
-	github.com/multiversx/mx-chain-core-go v1.2.9
+	github.com/multiversx/mx-chain-core-go v1.2.10-0.20230712104444-e818eaa01c7e
```
Comment (iulianpascalau) [inline go.mod:8]: proper releases

File: process/elasticproc/miniblocks/miniblocksProcessor.go [hunk 2]
```diff
@@ -121,6 +121,11 @@ func (mp *miniblocksProcessor) setFieldsMBIntraShardAndCrossFromMe(
 	case constructionState == int32(block.Final) && processingType == block.Processed.String():
 		dbMiniblock.ReceiverBlockHash = headerHash
 		dbMiniblock.ProcessingTypeOnDestination = processingType
+	case constructionState == 0 && processingType == "":
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply process/elasticproc/miniblocks/miniblocksProcessor.go:124]: yes, also the string is magic :)

File: go.mod (hunk 1 above)
Comment (miiu96) [reply go.mod:8]: will add

File: process/elasticproc/miniblocks/miniblocksProcessor.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (miiu96) [reply process/elasticproc/miniblocks/miniblocksProcessor.go:124]: changed the logic and removed that case
now the logic is: in the case of a miniblock without a miniblock header the processing type will be `Normal` and  the status `Final`

**Review by bogdan-rosianu — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]

**Review by sstanculeanu — APPROVED** [review]

**Pull request merged**